rule-extractor/pom.xml

```diff
@@ -59,6 +60,10 @@
       <artifactId>mockito-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Why do you need this dependency? If you really need it, please put it with other production dependencies, not with test dependencies.

Reply:
It was needed for the new `SecurityStandards` class, to use `ImmutableMap` and `Ordering.` Moved it under regular dependencies

Reply:
Normally, with Java 11+, you can avoid using Guava, it will allow to keep SLCORE distribution smaller